sources/engine/Stride.BepuPhysics/Stride.BepuPhysics/Constraints/BallSocketConstraintComponent.cs

```diff
@@ -57,6 +83,10 @@ public float SpringFrequency
         }
     }
 
+    /// <summary>
```

Comment:
@Eideren, updated, could you review this again? If it's ok I would add it to all other constraints in this PR.

Reply:
Yep, looks good too